src/main/java/graphql/testrunner/dto/Job.java

```diff
@@ -0,0 +1,38 @@
+package graphql.testrunner.dto;
+
+import java.util.List;
+import java.util.UUID;
+
+public class Job {
+
+    private UUID jobId;
```

Comment:
for what it's worth: If the class doesn't have any logic in it, I usually don't bother to protect the variables like this. This is really more of a data structure than a class. IE I would just have the three public variables. 

this is arguably more correct so no need to change it. more an "art of programming" thing